crates/anvil-polkadot/src/config.rs

```diff
@@ -407,7 +407,7 @@ Base Fee
 
 {}
 "#,
-            self.get_base_fee().green()
+            self.get_base_fee().saturating_mul_int(NATIVE_TO_ETH_RATIO as u128).green()
```

Comment:
instead of constantly multiplying this, why don't we keep it as an eth price and only convert it once by dividing it in `From<&'a AnvilNodeConfig> for GenesisConfig`

Reply:
[ababb58](https://github.com/paritytech/foundry-polkadot/pull/378/commits/ababb580fb5d3cc0ea8974d5c4787e3f90fb4863)